Example/PageBuilderExtensionSlider/view/adminhtml/ui_component/pagebuilder_slider_form.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd" extends="pagebuilder_base_form">
```

Comment:
So just making sure that autoplay and all of the base slider configuration in (Magento/PageBuilder/view/adminhtml/ui_component/pagebuilder_slider_form.xml) is viewable/editable (if that is the intention)?  I wouldn't know how it would be customizable without extending the original slider component configuration directly here.  Are you able to clarify how this config gets integrated with original component?

Reply:
This is extending a content type so we use the XML merging based on file name 😌

Reply:
Yes, the docs for this will explain that.